pkg/controller/plan/validation.go

```diff
@@ -143,6 +146,28 @@ func (r *Reconciler) validate(plan *api.Plan) error {
 		return err
 	}
 
+	// Validate version only if target ocp-to-ocp
```

Comment:
what did you mean by "if target ocp-to-ocp", maybe "Validate version only in case of openshift-to-openshift"?